src/main/java/catan/controllers/GameController.java

```diff
@@ -46,12 +46,19 @@ public GameIdDetails createNewGame(@RequestParam(value = "token", required = fal
                                             @RequestParam(value = "token", required = false) String token) throws AuthenticationException, GameException, WrongPathException {
 
         List<GameBean> games;
+        UserBean user;
+        int userId = 0;
```

Comment:
don't thinl that we need this variable, we can get value just via accessing the getter